lib/string/strdup/strndup.c

```diff
@@ -0,0 +1,7 @@
+// SPDX-FileCopyrightText: 2025, Alejandro Colomar <alx@kernel.org>
+// SPDX-License-Identifier: BSD-3-Clause
+
+
+#include <config.h>
+
+#include "string/strdup/strndup.h"
```

Comment:
Why do we need this module?

I cannot see any function defined here. And module which use macro `STRNDUP` already includes the header file `string/strdup/strndup.h`